pkg/cmd/server/api/types.go

```diff
@@ -399,8 +399,9 @@ type MasterConfig struct {
 	// AuditConfig holds information related to auditing capabilities.
 	AuditConfig AuditConfig
 
-	// EnableTemplateServiceBroker is a temporary switch which enables TemplateServiceBroker.
-	EnableTemplateServiceBroker bool
+	// TemplateServiceBrokerConfig holds information related to the template
+	// service broker
+	TemplateServiceBrokerConfig TemplateServiceBrokerConfig
```

Comment:
per my comments elsewhere, I'm not a fan of making people destroy their configuration to disable the feature.